### ISSUE TITLE
Henchbot fails to create deploy PRs for BinderHub updates

### ISSUE THREAD

**Issue by consideRatio** [henchbot/mybinder.org-upgrades#7]
As reported by @betatim in https://github.com/jupyterhub/binderhub/pull/1247#issuecomment-765900230, the henchbot doesn't seem to create PRs at the moment.

I guess it could be related to https://github.com/jupyterhub/chartpress/pull/115 which makes chartpress to write back strings to values.yaml with the same string formatting regarding no/single/double quotes it read a string initially with.

So, example:

```yaml
# input to chartpress
hub:
  image:
    tag: "some-string-to-be-replaced"
    
singleuser:
  image:
    tag: 'some-other-string-to-be-replaced'

proxy:
  image:
    tag: some-other-string-to-be-replaced
```

If chartpress loads this, and then writes back updated strings, their format will be the same as it was before now. But, previously, it would always be single quotes no matter what for the image tag's specifically.

```yaml
# output from chartpress 1.0.6
hub:
  image:
    tag: "updated-tag"
    
singleuser:
  image:
    tag: 'updated-tag'

proxy:
  image:
    tag: updated-tag
```

```yaml
# output from chartpress 1.0.0 to 1.0.5
hub:
  image:
    tag: 'updated-tag'
    
singleuser:
  image:
    tag: 'updated-tag'
    
proxy:
  image:
    tag: 'updated-tag'
```

**Comment by betatim** [comment]
The problem doesn't seem to be henchbot. I took a look at what the latest published chart is on https://jupyterhub.github.io/helm-chart/#development-releases-binderhub. There the latest chart is from 9 January 2021. This is "too old", we have merged changes to BinderHub since then.

Then I went and looked at the publish CI job for the last merge to master in BinderHub: https://github.com/jupyterhub/binderhub/runs/1749855591?check_suite_focus=true For some reason chartpress does not compute the correct version and hence concludes that the chart has already been published. Sounds like a bug n chartpress more than henchbot :-/
